### PR TITLE
Additional boot ops

### PR DESCRIPTION
**Pull request by kabir**
https://issues.redhat.com/browse/WFCORE-4770

### CODE REVIEW

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                throw rtex;
+            }
+            throw new RuntimeException(ex);
+        } finally {
+            if (ctx != null) {
+                ctx.terminateSession();
+            }
+            clearProperties();
+        }
+        LOGGER.info("Done executing CLI Script invoker for file " + file);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: info log must be i18ned

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    private static void processFile(File file, final CommandContext cmdCtx) throws IOException {
+        try ( BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
+            while (cmdCtx.getExitCode() == 0 && !cmdCtx.isTerminated() && line != null) {
+                LOGGER.debug("Executing command " + line.trim());
+                cmdCtx.handle(line.trim());
+                line = reader.readLine();
+            }
+        } catch (Throwable e) {
+            LOGGER.error("Unexpected exception processing commands ", e);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: error log must be i18ned

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+            throw new IllegalStateException("Failed to process file '" + file.getAbsolutePath() + "'", e);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: exception message must be i18ned

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        }
+        String warnFile = WildFlySecurityManager.getPropertyPrivileged("org.wildfly.internal.cli.boot.hook.script.warn.file", null);
+        if (warnFile != null) {
+            File warns = new File(warnFile);
+            if (warns.exists()) {
+                for (String line : Files.readAllLines(warns.toPath())) {
+                    LOGGER.warn(line);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: same remark as above

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+                }
+            }
+
+        }
+        String errorFile = WildFlySecurityManager.getPropertyPrivileged("org.wildfly.internal.cli.boot.hook.script.error.file", null);
+        if (errorFile != null) {
+            File errors = new File(errorFile);
+            if (errors.exists()) {
+                LOGGER.error("Error applying " + file + " CLI script. The Operations were executed but "
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: same remark as above

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+                        + "there were unexpected values. See list of errors in " + errors);
+                for (String line : Files.readAllLines(errors.toPath())) {
+                    LOGGER.error(line);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: same remark as above

File: cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+                }
+            }
+
+        }
+        if (cmdCtx.getExitCode() != 0 || cmdCtx.isTerminated()) {
+            throw new RuntimeException("Error applying " + file + " CLI script.");
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java]: same remark as above

File: cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java [hunk 8]
```diff
@@ -445,6 +449,44 @@
...
+        aeshCommands = new AeshCommands(this, new OperationCommandContainer(this));
+        this.cmdRegistry = aeshCommands.getRegistry();
+        this.cmdCompleter = null;
+        this.legacyCmdCompleter = null;
+        this.operationCandidatesProvider = null;
+
+        try {
+            initCommands(true);
+        } catch (CommandLineException | CommandLineParserException e) {
+            throw new CliInitializationException("Failed to initialize commands", e);
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java]: same remark as above

File: cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java [hunk 9]
```diff
@@ -523,18 +565,26 @@ private void restoreStdIO() {
...
     private void initCommands() throws CommandLineException, CommandLineParserException {
+        initCommands(false);
+    }
+
+    private void initCommands(boolean bootInvoker) throws CommandLineException, CommandLineParserException {
         // aesh commands
         cmdRegistry.addCommand(new VersionCommand());
         cmdRegistry.addCommand(new HelpCommand(cmdRegistry));
-        cmdRegistry.addCommand(new ConnectCommand());
+        if (!bootInvoker) {
```
Comment (jmesnil) [inline cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java]: stylistic suggestion: maybe you should add things that change with `bootInvoker` in a single `if (!bootInvoker)`?

Comment (jfdenise) [reply cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java]: @jmesnil I would prefer to keep the current order when registering commands.

**Comment by jfdenise** [comment]
@jmesnil , logging fixed.

**Comment by jmesnil** [comment]
@kabir @jfdenise thanks!

**Pull request merged**